man/networkctl.xml

```diff
@@ -495,6 +495,21 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
 
         <xi:include href="version-info.xml" xpointer="v256"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>
+          <command>persistent-storage</command>
+          <replaceable>BOOL</replaceable>
+        </term>
+        <listitem>
+          <para>Notify <filename>systemd-networkd.service</filename> if the persistent storage for the
+          service is ready. This is called by
+          <filename>systemd-networkd-persistent-storage.service</filename>, and is not necessary to be called
+          by users or administrators.</para>
```

Comment:
Hmm, if this is the case, maybe we should just not document it? This could still give wrong impressions to users if they only check --help, and as said it's generally not expected that it shall be called outside of the service.

Reply:
Or even make this an `invoked_as` thing?

Reply:
`switch-root` is pretty much generic. It takes a path although we assume `/sysroot/` across our code. People who want to use a different set of initrd units could and do make use of it (I once saw people using it for achieving something similar to `soft-reboot`. There's a reason why we restricted switch-root to be used in initrd only _after_ `soft-reboot` was introduced)

This, however, only controls an internal state of networkd. And I really cannot think of a case where people would call it behind our back.

> Otherwise, people needs to read code to understand what the command does.

Hmm, we have tools like `systemd-creds`, that when invoked through socket activation, automatically sets up a varlink server. This is nowhere similar to their CLI usage.

Reply:
That said, that's only an opinion. I'm not strong against this if you want to do it anyway :)

Reply:
If not only you suggest that we should hide the command from the document, then I think we should use varlinkctl in the .service unit in this case, instead of introducing such tiny wrapper of the varlink command.